src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -1557,6 +1557,32 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
                 "0",
                 "0",
                 "Harry Potter"
+            },
+            new Object[]{
+                HEARING_REMINDER_NOTIFICATION,
+                "oral",
+                Collections.singletonList("774a5cba-fab6-4b8c-a9d9-03f913ed2dca"),
+                Collections.singletonList("404e9a43-6318-492c-b5c2-e34ddfbbdde9"),
+                Collections.emptyList(),
+                "yes",
+                "yes",
+                "1",
+                "1",
+                "0",
+                "Appointee Appointee"
+            },
+            new Object[]{
+                HEARING_REMINDER_NOTIFICATION,
+                "oral",
+                Collections.emptyList(),
+                Collections.emptyList(),
+                Collections.emptyList(),
+                "no",
+                "no",
+                "0",
+                "0",
+                "0",
+                ""
```

Comment:
I paper hearing Type scenario to cover the hearingReminder here would be good as well.